### PR TITLE
Fix package-lock.json for lisk-transactions - Closes #1060

### LINKED ISSUE #1060: Fix package-lock.json for lisk-transactions
### Expected behavior

No errors when running `npm i`

### Actual behavior

Error `stack Error: 404 Not Found: har-validator@5.1.2` when running `npm i`

### Steps to reproduce

- clone the repository
- switch to development branch
- run npm i

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

Error stack Error: 404 Not Found: har-validator@5.1.2 when running npm i

### How did I fix it?

By updating package.json and package-lock.json

### How to test it?

`npm i` should not throw errors

### Review checklist

* The PR resolves #1060
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,5 +53,7 @@
 		"tslint-immutable": "4.7.0",
 		"typescript": "3.1.1"
 	},
-	"dependencies": {}
+	"dependencies": {
```
Comment (shuse2) [inline package.json]: u don't need to add the dependency here ='(

File: package-lock.json [hunk 2]
```diff
@@ -730,15 +730,14 @@
 			}
 		},
 		"ajv": {
-			"version": "5.5.2",
-			"resolved": "https://registry.npmjs.org/ajv/-/ajv-5.5.2.tgz",
-			"integrity": "sha1-c7Xuyj+rZT49P5Qis0GtQiBdyWU=",
-			"dev": true,
+			"version": "6.7.0",
+			"resolved": "https://registry.npmjs.org/ajv/-/ajv-6.7.0.tgz",
```
Comment (shuse2) [inline package-lock.json]: im not sure this change is because of the added dependency in the `package.json` or it was meant to be updated 😕

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (pablitovicente) [reply package.json]: Corrected the commit

File: package-lock.json (hunk 2 above)
Comment (pablitovicente) [reply package-lock.json]: Corrected the commit

**Review by shuse2 — APPROVED** [review]

**Pull request merged**